src/App.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <h1>LikeCoin NFT Dashboard</h1>
+  <div>
+    <div>Select a page</div>
+    <div>
+      <span v-for="(route, i) of routes" :key="i">
```

Comment:
Conceptually key should better be `route.name` (although for static amount of pages i would work too...)

Reply:
Stole these code from nnkken 🌚